scripting/2.0/modules/ROOT/pages/edits/scripting-module-studio.adoc

```diff
@@ -0,0 +1,234 @@
+
+
+[[add-connector-to-project]]
+== Add the Module to Your Mule Project
+
+Add Scripting Module to a Mule project to automatically populate the XML code with the module's namespace and schema location, as well as add the required dependencies to the project's `pom.xml` file:
+
+. In the *Mule Palette* view, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type `Scripting Module` in the search field.
+. Click *Scripting Module* in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+Adding a module to a Mule project in Studio does not make that module available to other projects in your Studio workspace.
+
+
+[[configure-input-source]]
+== Configure an Input Source
+
+An input source initiates a flow when a specified condition is met.
+You can configure one of these input sources to use with the Scripting Module:
+
+* `HTTP Listener`, which initiates a flow each time it receives a request on the configured host and port
+* `Scheduler`, which initiates a flow when a time-based condition is met
+
+For example, to configure `HTTP Listener`, follow these steps:
+
+. In the *Mule Palette* view, select *HTTP* > *Listener*.
+. Drag *Listener* to the Studio canvas.
+. On the *Listener* configuration screen, optionally change the value of the *Display Name* field.
+. Specify a value for the *Path* field.
+. Click the plus sign (+) next to the *Connector configuration* field to configure a global element that can be used by all instances of `HTTP Listener` in the app.
+. On the *General* tab, specify connection information.
+. On the *TLS* tab, optionally specify TLS information.
+. On the *Advanced* tab, optionally specify reconnection information, including a reconnection strategy.
+. Click *Test Connection* to confirm that Mule can connect with the specific server.
+. Click *OK*.
+
+[[add-connector-operation]]
+== Add Execute Operation to the Flow
+
+When you add a module operation to your flow, you immediately define a specific operation for that module to perform.
+
+To add the Execute operation for Scripting Module, follow these steps:
+
+. In the *Mule Palette* view, select *Scripting Module* and then select the Execute operation.
+. Drag the *Execute* operation onto the Studio canvas to the right of the input source.
+. Click the *Execute* operation to open the Execute configuration tab.
+. Use one of the following two options to install and configure a `jsr-223` compliant engine: +
+
+* In the *Required Libraries* section, click *Configure...* and select one of the following options to search and install the engine: +
+
+** Add recommended libraries +
+Install Groovy engine as recommended. This option only appears if you did not previously install Groovy.
+** Use local file +
+Browse a local file for the required engine library and install it.
+** Add Maven dependency +
+Browse the engine dependency and install it. +
+
+* In the `pom.xml` file, manually add the engine in both of these sections:
```

Comment:
Are you adding the engine to the TAG or to the SECTION in the following two sentences?